helm/hpcc/values.yaml

```diff
@@ -509,6 +517,8 @@ roxie:
     listenQueue: 200
     numThreads: 30
     visibility: local
+    # Can override ingress rules for each service if desired - for example to add no additional ingress permissions you can use
+    # ingress: []
```

Comment:
I wonder if there should be an option to merge with the global ingress rules.

Reply:
Because they are lists rather than dictionaries, it's not clear what merging would mean. In particular I suspect it would mean "allow whatever either location says is allowed" which means you would not be able to override with something MORE restrictive.